src/orange/MatrixUtils.cc

```diff
@@ -73,7 +74,7 @@ gemm(SquareMatrix<T, N> const& a, SquareMatrix<T, N> const& b)
             // Accumulate dot products
             for (size_type k = 0; k != N; ++k)
             {
-                result[i][j] = std::fma(b[k][j], a[i][k], result[i][j]);
+                result[i][j] = fma(b[k][j], a[i][k], result[i][j]);
```

Comment:
There are other usage that should be change:
```
src/orange/MatrixUtils.hh:            result[i] = std::fma(alpha, a[i][j] * x[j], result[i]);
src/orange/MatrixUtils.hh:            result[i] = std::fma(alpha, a[j][i] * x[j], result[i]);
```
and some that should be checked and/or just changed for consistency:
```
src/corecel/grid/Interpolator.hh:        std::fma(slope_, XTraits_t::add_transformed(offset_, x), intercept_));
src/celeritas/grid/PolyEvaluator.hh:        return std::fma(arg, calc_impl<M + 1>(arg), coeffs_[M]);
src/celeritas/random/distribution/UniformRealDistribution.hh:    return std::fma(delta_, generate_canonical<RealType>(rng), a_);
src/celeritas/random/distribution/NormalDistribution.hh:        return std::fma(spare_, stddev_, mean_);
src/celeritas/random/distribution/NormalDistribution.hh:    return std::fma(r * std::sin(theta), stddev_, mean_);
src/corecel/math/ArrayUtils.hh:        (*y)[i] = std::fma(a, x[i], (*y)[i]);
src/corecel/math/ArrayUtils.hh:        result = std::fma(x[i], y[i], result);
```

Reply:
> but the floating-point-only classes we shouldn't have to. 

I agree we don't have to and I am proposing those to be consistent (no need to 'think' which one to use) and future proofing (if the function becomes a template for whatever reason).